app/services/classroom_config.rb

```diff
@@ -47,8 +48,65 @@ def configurable?(repo)
   # Returns nothing
   def generate_issues(repo, tree_sha)
     @github_repository.tree_objects(tree_sha).each do |issue|
-      blob = @github_repository.blob(issue.sha)
-      repo.create_issue(blob.data["title"], blob.body) if blob.data.present?
+      blob = formated_blob(@github_repository.blob(issue.sha))
+      next if blob.title.blank?
+      repo.create_issue(blob.title, blob.body, labels: blob.labels.present? ? blob.labels : [])
```

Comment:
In general, we can refactor ternaries that check for presentness/blankness to use the `||` operator.

`labels: (blob.labels || [])`